test/cypress/support/deployments.ts

```diff
@@ -50,6 +50,11 @@ const commands = {
             cy.get('.input input').clear().type(deploymentId);
             cy.get('.input.loading').should('not.exist');
         });
+    },
+    revertToDefaultValue: () => {
+        const revertToDefaultAriaLabel = '[aria-label="revert-to-default"]';
+        cy.get(revertToDefaultAriaLabel).click();
+        cy.get(revertToDefaultAriaLabel).should('not.exist');
```

Comment:
Optional:

I have recently learned that this can be a single like, like done in https://docs.cypress.io/guides/core-concepts/introduction-to-cypress.html#Example-2-Reversing-the-Default-Assertion

```suggestion
        cy.get(revertToDefaultAriaLabel).click().should('not.exist');
```

Not sure if it's better, but I'm leaving it out here for your consideration :slightly_smiling_face:

Reply:
Checked, works equally well, so changed.